script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -278,48 +278,48 @@ I see―――I'm shocked to hear he's still alive.
 [sha:fbe278504ff56ce8d4e3b24548902f7503da224b]{
 -- Page 94, Offset 30807.
 -- どうやら、そちらはそちらで愉快な<事|こ><態|と>になっているな？」
-If that's the case, things must be getting pretty interesting over there."
+It appears things taken an amusing turn for you people, in its own way."
 }
 [sha:f514dcdd128ee699f64ac12832197c6d5c2c8d16]{
 -- Page 95, Offset 30808.
 -- 「“ああ、おかげさまでな、爺さんは今も正気じゃねえよ。
-"Yeah, thanks to you. The old man's losing his mind even as we speak.
+"Yeah, thanks to you. The ol' man's losin* his mind even as we speak.
 }
 [sha:00aa87378b996f210783e59ddf21eb0267da5a34]{
 -- Page 95, Offset 30809.
 -- ……で、どうなんだ？　テメェ、完全にバルダムヨォンか？
-...And how about you? Are you completely Valdamjong now?
+So, how's it goin'? You completely Valdamjong now?
 }
 [sha:884af5c392b51d629551fa3a63da734f8182d455]{
 -- Page 95, Offset 30810.
 -- 転生は成功したんだろうな？　まだだったら遠慮はいらねえ、あと七日は教会からの支援は止めておくから、さっさと目を覚ましてきな”」
-Your reincarnation was a success, right? If not, then get on with it.
- I can only hold off the Church for seven days, so come on and wake up already."
+Reincarnation complete, right? If nah, then don't hold back.
+ Can only hold off the Church for 'bout seven days, so get a move on, will ya?"
 }
 [sha:7ea7da4bfc4d4755543195f816bd6c2f2238c18e]{
 -- Page 95, Offset 30811.
 -- 「……<改|あらた>めなさい司祭代行。それは教会への背信行為です」
-"...Choose your words carefully, Acting Presbyter.
+"Choose your words carefully, Acting Presbyter...
  That could be considered treason against the Church."
 }
 [sha:e59d6ec4346be0065cef0c0f36c091df6a0a5460]{
 -- Page 96, Offset 30812.
 -- 「“ほざけ、テメェに言われたくはねぇよシエル。
-"Piss off. I don't want to hear that from you, Ciel.
+"Fuck off. Don't wanna that shit from you, Ciel.
```

Comment:
want that shit

Reply:
Think I just forgot "hear" for some reason.